deb-get

```diff
@@ -2413,6 +2413,17 @@ function deb_mullvad-vpn() {
     SUMMARY="VPN Service based in Sweden that does not collect user data."
 }
 
+function deb_xournalpp() {
+    get_github_releases "https://api.github.com/repos/xournalpp/xournalpp/releases/latest"
+    if [ "${ACTION}" != "prettylist" ]; then
+        URL="$(grep "browser_download_url.*.deb" "${CACHE_DIR}/${APP}.json" | head -n1 | cut -d'"' -f4)"
```

Comment:
Multiple files are provided, depending on the version of Debian/Ubuntu. The correct file should be selected based on `UPSTREAM_CODENAME`, and the supported codenames should be listed in `CODENAMES_SUPPORTED`. Read [CONTRIBUTING#creating-the-function](https://github.com/wimpysworld/deb-get/blob/main/CONTRIBUTING.md#creating-the-function) for further information.